modules/platforms/dotnet/Apache.Ignite.Core.Tests/TestUtils.cs

```diff
@@ -730,4 +730,13 @@ public bool IsEnabled(LogLevel level)
             }
         }
     }
+
+    /// <summary>
+    /// Interop class.
+    /// </summary>
+    public class PlatformComputeBinarizable
```

Comment:
Let's move it to a separate file, `TestUtils` is already quite long.

Reply:
Fixed.